docs/production/podman.rst

```diff
@@ -0,0 +1,154 @@
+.. _podman:
+
+Podman
+======
+
+Contrary to the Containers section, this Podman manual is targeted at small instances.
+It uses SQLite as a database and locally cloned source repository.
+
+.. note::
+
+	This tutorial supports multiple bot instances running under the same system account.
+	When following it, replace $BOTNAME with the name of your bot (or use generic ``pumpkin`` name, just be consistent).
+	Make sure it does not contain spaces, slashes or other funny characters.
+
+
+.. _podman_download:
+
+Fetching source code
+--------------------
+
+Use ``git`` to download the source code.
+
+.. warning::
+	It is necessary to use HTTPS, because the container should not have access to your personal SSH keys.
+	If you need to clone a private repository, you can use GitHub's Personal :ref:`access_tokens` with REPO:READ permissions.
+
+.. code-block:: bash
+
+	git clone https://github.com/pumpkin-py/pumpkin-py.git $BOTNAME
+	cd $BOTNAME
+
+To update the bot later, run
+
+.. code-block:: bash
+
+	cd $BOTNAME
+	git pull
+
+
+.. _podman_token:
+
+Discord bot token
+-----------------
+
+See :ref:`general_token` in chapter General Bot Information.
+
+
+.. _podman_image:
+
+Creating Pumpkin image
+----------------------
+
+.. code-block:: bash
+
+    podman build --file Dockerfile --tag pumpkin-py
+    # and do the following for all the bots you host like this
+    podman tag pumpkin-py:latest $BOTNAME
+
+.. _podman_env_file:
+
+The ``.env`` file
+-----------------
+
+The environment file contains variables necessary for the bot to function.
+
+.. code-block:: bash
+
+    # A string passed to SQLAlchemy
+    DB_STRING=sqlite:///pumpkin-py/pumpkin.db
```

Comment:
I know this sounds stupid, but from my testing, one additional `/` is missing here. The sqlite uses 3 slashes for the prefix and one additional is needed as the filesystem root dir. Other functioning solution would be `sqlite:///pumpkin.db`
```suggestion
    DB_STRING=sqlite:////pumpkin-py/pumpkin.db
```